src/aioswitcher/device/tools.py

```diff
@@ -234,13 +238,20 @@ async def validate_token(username: str, token: str) -> bool:
 #   used in retreving state from the packet.
 # Used for Switcher Runners and Switcher Lights
 # Runner and Runner Mini: has no lights circuits & one shutter circuits ->
-#   shutter circuit is 0, get_light_discovery_packet_index would raise an error.
+#   shutter circuit is 0,
+#   get_light_discovery_packet_index would raise an error.
```

Comment:
Why breaking these lines? they are less than 80 chars, and more readable without breaking into two lines

Reply:
I had to ... because of linters said too long so I want too all those lines look the same.

Reply:
So fix the long lines, I don't see why we need to make a long discussion for every simple review comment. Try avoiding changing lines that are not needed for the PR.

Reply:
It is done.